supertokens_python/recipe/multitenancy/interfaces.py

```diff
@@ -101,7 +101,7 @@ def __init__(
         emailpassword: EmailPasswordConfig,
         passwordless: PasswordlessConfig,
         third_party: ThirdPartyConfig,
-        core_config: Dict[str, Any],
+        core_config: Optional[Dict[str, Any]],
```

Comment:
this should not be changed optional... you can make a different type for this in the dashboard recipe which will NOT have this core_config prop and use that in the dashboard recipe. But do not change the mulittenancy recipe at all.

Reply:
DOne